sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/README.md

```diff
@@ -0,0 +1,134 @@
+# TODO - Validate these instructions
```

Comment:
This file is not needed and also has wrong info in it :)

Reply:
I think that having a README file and an example is useful though (at least it was for me). Once the PR is complete we can clean up what is not needed or wrong, WDYT?